documentation/source/newsfragments/2658.removal.rst

```diff
@@ -0,0 +1 @@
+Deprecate ``<=`` syntax for setting values on handles.
```

Comment:
Please indicate the alternative and the motivation; users reading this will want to know more.

Reply:
Will the link to the issue which is generated by towncrier be sufficient? #2658. Will add the alternative for sure.

Reply:
#2658 feels to me like the final nail in the coffin. Maybe copy-paste the list of reasons that came to my mind in https://github.com/cocotb/cocotb/pull/2667#pullrequestreview-734467878?

Reply:
Multiline newsfragments are not supported by default by towncrier. If included it should be succinct, or I could change the link to a dedicated issue, or this PR?

Reply:
How about something like:

" Deprecate ``signal <= newval`` syntax for setting values on handles for being unreliable in corner cases and making cocotb harder to learn and teach. Use ``signal.value <= newval`` instead."